CHANGELOG.md

```diff
@@ -2,9 +2,11 @@
 
 # Unreleased
 
-- Add `ocaml.copy-type-under-cursor` to copy, in the clipboard the type of 
+- Add `ocaml.copy-type-under-cursor` to copy, in the clipboard the type of
   the expression under the cursor (#1582)
 
+- Add modules for `ocaml-lsp` custom requests to copy (#1619)
```

Comment:
So far there is no user face changes in that PR.

Reply:
Yeah this was to avoid the ci from failing since I can't add a `no changelog` label